edflow/data/believers/meta.py

```diff
@@ -229,7 +229,7 @@ def load_labels(root):
     return labels
 
 
-def clean_keys(labels):
+def clean_keys(labels, loaders):
     """Removes all loader inforamtion from the keys.
```

Comment:
inforamtion -> information
you can decide if you want to fix it before merging :)